src/v/features/feature_table.cc

```diff
@@ -423,6 +423,20 @@ void feature_table::testing_activate_all() {
     on_update();
 }
 
+feature_table::version_fence
```

Comment:
should commit message start from `features:` ?

Reply:
i went back and forth on it. changed to `features:`